modules/core/src/main/scala/org/scalasteward/core/sbt/SbtAlg.scala

```diff
@@ -110,8 +111,10 @@ object SbtAlg {
           maybeClearCredentials = if (config.keepCredentials) Nil else List(setCredentialsToNil)
           commands = maybeClearCredentials ++
             List(dependencyUpdates, reloadPlugins, dependencyUpdates)
-          lines <- exec(sbtCmd(commands), repoDir)
-        } yield parser.parseSingleUpdates(lines)
+          sourceUpdates = exec(sbtCmd(commands), repoDir).map(parser.parseSingleUpdates)
+          buildPropUpdates = proposeBuildPropertiesUpdate()
+          updates <- (sourceUpdates product buildPropUpdates).map(p => p._1 ::: p._2)
```

Comment:
I do not know how to concate two `F[List[A]]` in `cats` so this concatenation may be funny.

Reply:
What about
```scala
  sourceUpdates <- ...
  buildPropUpdates <- ...
} yield sourceUpdates ++ buildPropUpdates
```
?

Reply:
Done in https://github.com/fthomas/scala-steward/pull/665/commits/200b7c6de1f389af6287ba90468aa1eb304cda1f